app/models/user.rb

```diff
@@ -179,6 +179,7 @@ def ssn_normalized
   delegate :mhv_correlation_id, to: :mpi, prefix: true
   delegate :mhv_ien, to: :mpi
   delegate :mhv_iens, to: :mpi, prefix: true
+  delegate :multiple_sec_ids?, to: :mpi
```

Comment:
I'm not sure we even care about `multiple_sec_ids?` on the user model, so I probably wouldn't even bother with it here